src/pcrlock/pehash.c

```diff
@@ -135,6 +135,10 @@ int pe_hash(int fd,
                 r = hash_file(fd, mdctx, p, st.st_size - p - certificate_table->Size);
                 if (r < 0)
                         return r;
+
+                /* If the file size is not a multiple of 8 bytes, pad the hash with zero bytes. */
+                if (st.st_size % 8 != 0 && EVP_DigestUpdate(mdctx, (const uint8_t[8]) {}, 8 - (st.st_size % 8)) != 1)
```

Comment:
btw, strictly speaking the array only needs to be 7 bytes long, not 8

Reply:
Right but that just makes the code harder to read if you ask me, a single extra byte isn't going to hurt anyone

Reply:
i mean, there's a reason i didn't set the minor-suggestions label even ;-)